templates/_base-images-ref.html

```diff
@@ -0,0 +1,461 @@
+<script src="//cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.15/lodash.min.js" type="text/javascript"></script>
+<script type="text/javascript">
+
+  (function () {
+    window.jQuery
+      .ajax({
+        url: "https://api.balena-cloud.com/v6/device_type?$select=name,slug&$expand=is_of__cpu_architecture($select=slug)&$filter=is_default_for__application/any(idfa:((idfa/is_host%20eq%20true)%20and%20(idfa/is_archived%20eq%20false))%20and%20not(contains(idfa/app_name,%27esr%27)))&$orderby=name%20asc"
+      })
+      .done(function (deviceTypes) {
+
+        var IGNORE_THESE_DEVICES = ['n310-tx2', 'apalis-imx6', 'colibri-imx6', 'cubox-i', 'edge', 'etcher-pro', 'edison', 'hummingboard2', 'odroid-u3+', 'odroid-ux3', 'nuc', 'parallella-hdmi-resin', 'solidrun-imx6']
+        var table = "";
+        var devices = {};
```

Comment:
```suggestion
        var deviceTypesByArch = {};
```
right?

Reply:
Renamed the variable.

I think just using `forEach` loop with the DTs list is better here since we can add some extra logic to exclude the device in ignore list or replace some contents in the callback. `_.groupBy` is fine but we need to add extra steps after grouping them to the above tasks.